borsh-ts/index.ts

```diff
@@ -319,6 +319,20 @@ function deserializeField(schema: Schema, fieldName: string, fieldType: any, rea
             return reader.readArray(() => deserializeField(schema, fieldName, fieldType[0], reader));
         }
 
+        if (fieldType.kind === 'option') {
+            const option = reader.readU8();
+            if (option) {
+                return deserializeField(
+                    schema,
+                    fieldName,
+                    fieldType.type,
+                    reader
+                );
+            }
+      
+            return undefined;
```

Comment:
Overall looks very neat! What is the reason of return undefined instead of null here?

Reply:
Thanks, undefined aligns better with default uninitialized field in JavaScript.

Reply:
Got it! I can merge your PR but I don't have access to release, so let me ping @volovyk-s

Reply:
Ok, I can publish it. @bartosz-lipinski , please revert the version back to `0.3.1`. It will be bumped by `np` later.

Reply:
Thanks a lot, @volovyk-s, I reverted the version and merged changes back from the master.